<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="buildBtn">
   <property name="geometry">
    <rect>
     <x>361</x>
     <y>7</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>построить дерево</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addBtn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delRepBtn">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>7</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>удалить повторяющиеся</string>
   </property>
  </widget>
  <widget class="QPushButton" name="timeBtn">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>40</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>замеры времени</string>
   </property>
  </widget>
  <widget class="QLabel" name="imgLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>641</width>
     <height>431</height>
    </rect>
   </property>
   <property name="text">
    <string>                                                                         TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="strEdt">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="delBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>удалить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>строка</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>буква</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="letEdt">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="outLbl">
   <property name="geometry">
    <rect>
     <x>337</x>
     <y>45</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
